packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/style/LinearGradient.kt

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.uimanager.style
+
+import android.graphics.Shader
+import android.graphics.LinearGradient as AndroidLinearGradient
+import com.facebook.react.bridge.ReadableMap
+import kotlin.math.atan
+import kotlin.math.tan
+
+internal class LinearGradient(
+  private val orientationMap: ReadableMap,
```

Comment:
It doesn't seem like we need to keep this around

```suggestion
  orientationMap: ReadableMap,
```